platformio.ini

```diff
@@ -21,6 +22,7 @@ board_build.partitions = huge_app.csv
 extends = env:atom_matrix
 build_flags =
   ${env:atom_matrix.build_flags}
+  '-UMAX_MW -DMAX_MW=4200'  # Reduce power limit to 4W for host-connected development boards
```

Comment:
nit: mention that this 4.2W number comes from the fact that the max in the USB 3.0 standard is 4.5W

Also, please do the same for the `esp8266_dim` config below. That or maybe even safer to add code somewhere like this:

```
#if START_DIM && MAX_NW
#  undef MAX_NW
#  define MAX_NW 4200
#endif
```